connector_jira_servicedesk/models/account_analytic_line/common.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import api, fields, models
+
+
+class JiraAccountAnalyticLine(models.Model):
+    _inherit = 'jira.account.analytic.line'
+
+    jira_servicedesk_issue_url = fields.Char(
+        string='Original JIRA issue Link',
+        compute='_compute_jira_servicedesk_issue_url',
+    )
+
+    @api.depends('jira_issue_key')
+    def _compute_jira_servicedesk_issue_url(self):
+        """Compute the external URL to JIRA service desk."""
+        for record in self:
+            jira_project = self.project_id.jira_bind_ids[0]
```

Comment:
Are you sure to always have a project and a binding for it?
If not, your code will can raise.